src/database/repositories/EventRepository.js

```diff
@@ -11,11 +11,11 @@ class EventRepository extends Repository {
     }
 
     async doesEventExistsWithName(name) {
-        const entity = await this.entity.findOne({ name: name, archived: false, deleted: false });
+        const entity = await this.entity.findOne({ name: name, deleted: false });
         return entity != null;
     }
 
-    async getEvents(archived = false, fields = null) {
+    async getEvents(archived = true, fields = null) {
```

Comment:
archived is default parameter!!!!!! use archived = false!!!!!!!!!!!!!!

Reply:
sxjbcv klmansc¨§